src/wallet/migrate.cpp

```diff
@@ -564,6 +564,24 @@ void BerkeleyRODatabase::Open()
         throw std::runtime_error("BDB builtin encryption is not supported");
     }
 
+    // Check all LSNs point to file 0 and offset 1 which indicates that the LSNs were
```

Comment:
Nit: Would be nice to say what LSN means, e.g. "Check all LSNs (log sequence numbers)..."

Reply:
Done